samples/bluetooth/fast_pair/locator_tag/Kconfig

```diff
@@ -8,16 +8,16 @@ source "Kconfig.zephyr"
 
 menu "Nordic Fast Pair Locator tag sample"
 
-choice FAST_PAIR_LOCATOR_TAG_PLATFORM
-	prompt "Locator tag sample platform"
-	default FAST_PAIR_LOCATOR_TAG_PLATFORM_THINGY if BOARD_THINGY53_NRF5340_CPUAPP
-	default FAST_PAIR_LOCATOR_TAG_PLATFORM_DK
+choice APP_PLATFORM
+	prompt "Application platform"
```

Comment:
Should it actually have prompts? There is no need to assign it by the user (property determined by the used board - a supported board could update default here I think).

Reply:
Actually maybe we can even rely directly on board Kconfig here?